registry/handlers/app.go

```diff
@@ -249,6 +250,14 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 			context.Repository = notifications.Listen(
 				repository,
 				app.eventBridge(context, r))
+
+			context.Repository, err = applyRepoMiddleware(context.Repository, app.Config.Middleware["repository"])
+			if err != nil {
+				context.Errors.Push(v2.ErrorCodeUnknown, err)
```

Comment:
Isn't it there?
https://github.com/docker/distribution/pull/244/files#diff-3ce751bb26687df9f572b431f7055afaR256

On Wed, Mar 11, 2015 at 2:01 PM Stephen Day <notifications@github.com>
wrote:

> In registry/handlers/app.go
> <https://github.com/docker/distribution/pull/244#discussion_r26238252>:
>
> > @@ -249,6 +250,14 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
> >  			context.Repository = notifications.Listen(
> >  				repository,
> >  				app.eventBridge(context, r))
> > +
> > +			context.Repository, err = applyRepoMiddleware(context.Repository, app.Config.Middleware["repository"])
> > +			if err != nil {
> > +				context.Errors.Push(v2.ErrorCodeUnknown, err)
>
> We still need a log message here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/distribution/pull/244/files#r26238252>.
>